Plots/Polygons/NCL_polyg_18.py

```diff
@@ -0,0 +1,133 @@
+"""
+NCL_polyg_18.py
+==============
+This script illustrates the following concepts:
+   - Adding lines, markers, and polygons to a map
+   - Drawing lines, markers, polygons, and text in inset axes
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/polyg_18.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/polyg_18_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+# ----------------
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy
+from geocat.viz import util as gvutil
+import matplotlib.patches as mpatches
+from cartopy.mpl.ticker import LatitudeFormatter, LongitudeFormatter
+import numpy as np
+###############################################################################
+# Define helper function to remove ticks/frames from axes
+
+
+def removeTicks(axis):
+    axis.get_xaxis().set_visible(False)
+    axis.get_yaxis().set_visible(False)
+
+
+###############################################################################
+# Plot map, markers, and polygons
+
+# Set size of figure
+fig = plt.figure(figsize=(10, 10))
+
+# Make grid on figure with 2 rows, 1 column
+grid = plt.GridSpec(2, 20, hspace=0.2, wspace=0.2, figure=fig)
+
+# Make subplot for map
+ax = plt.subplot(grid[:-1, 1:], projection=ccrs.PlateCarree())
+
+# Add continents
+continents = cartopy.feature.NaturalEarthFeature(
+        name="coastline",
+        category="physical",
+        scale="50m",
+        edgecolor="None",
+        facecolor="lightgray",
+)
+ax.add_feature(continents)
+
+# Set map extent
+ax.set_extent([-180, 180, -90, 90], crs=ccrs.PlateCarree())
+
+# Create arrays with location and design of each marker
+lon = np.arange(-160, 160, 20)
+lat = np.arange(-80, 80, 10)
+marker = ['.', '+', '*', 'o', 'x', 's', '^', 'v', 'D', '>', '<', 'p', 'h', '8', 'X', 'd']
+
+# Draw markers on diagonal line across graph
+for x in range(len(lon)):
+    ax.plot(lon[x], lat[x], marker=marker[x], color='blue', fillstyle='none', markersize=18, zorder=3)
+
+# Draw small red box in upper center
+ax.add_patch(mpatches.Rectangle(xy=[7, 47], width=9, height=7, facecolor='None', edgecolor='red', alpha=1.0, transform=ccrs.PlateCarree(), zorder=5))
+
+# Draw green window in bottom right
+ax.add_patch(mpatches.Rectangle(xy=[110, -45], width=50, height=35, facecolor='lime', alpha=0.3, transform=ccrs.PlateCarree(), zorder=5))
+
+# Use gvutil function to set the ticks on axes
+gvutil.set_axes_limits_and_ticks(ax, xlim=None, ylim=None, xticks=[-180, -150, -120, -90, -60, -30, 0, 30, 60, 90, 120, 150, 180], yticks=[-90, -60, -30, 0, 30, 60, 90], xticklabels=None, yticklabels=None)
```

Comment:
To specify tick marks, use `np.arrange`. This method is concise and it makes code more readable.